cs251tk/toolkit/__main__.py

```diff
@@ -45,9 +45,28 @@ def run_server(basedir):
     return
 
 
+def download_specs(course, basedir, stogit):
+    spec_urls = {
+        'sd': 'https://github.com/StoDevX/cs251-specs.git',
+        'hd': 'https://github.com/StoDevX/cs241-specs.git',
+        'ads': 'https://github.com/Jedmeyer/cs253-specs.git'
+    }
+    course = course.split("-")[0].lower()
+    try:
+        url = spec_urls[course]
+    except KeyError:
+        print("Course {} not recognized".format(course))
+        sys.exit(1)
+    with chdir(basedir):
+        run(['git', 'clone', url, 'data'])
+        if not stogit:
+            return compute_stogit_url(course="hd", stogit=None, _now=datetime.date.today())
```

Comment:
Is this supposed to be hard-coded to Hardware?

Reply:
Nope. That's left over from copying the last version of it.